src/Smd.php

```diff
@@ -250,18 +260,18 @@ public function getId()
      * @param  string $description
      * @return self
      */
-    public function setDescription($description)
+    public function setDescription(string $description) : self
     {
-        $this->description = (string) $description;
-        return $this->description;
```

Comment:
I think this may be considered as a bug in the current version.
We would need a bugfix for it (PR into master).

Reply:
Noted. I will open an issue.